doc/dependencies.md

```diff
@@ -7,7 +7,7 @@ These are the dependencies currently used by Bitcoin Core. You can find instruct
 | --- | --- | --- | --- | --- | --- |
 | Berkeley DB | [4.8.30](https://www.oracle.com/technetwork/database/database-technologies/berkeleydb/downloads/index.html) | 4.8.x | No |  |  |
 | Boost | [1.77.0](https://www.boost.org/users/download/) | [1.64.0](https://github.com/bitcoin/bitcoin/pull/22320) | No |  |  |
-| Clang<sup>[ \* ](#note1)</sup> |  | [7.0](https://releases.llvm.org/download.html) (C++17 & std::filesystem support) |  |  |  |
+| Clang |  | [8.0](https://releases.llvm.org/download.html) (C++17 & std::filesystem support) |  |  |  |
```

Comment:
```suggestion
| Clang |  | [8.0](https://releases.llvm.org/download.html) (C++17, std::filesystem, and P0083R3 support) |  |  |  |
```

Reply:
I don't think mentioning a paper in the dependencies will give users any useful information. If they are interested in this, they can use https://en.cppreference.com/w/cpp/compiler_support or the documentation of their compiler. Practically they should only care about the minimum required version.